Added separate Contact Us form with validation : Issue no #231

AbhijitMotekar99 opened the pull request:
### Solved :
Issue no #231 


![Yarn contact us](https://github.com/user-attachments/assets/9ebaedb8-014f-4304-814e-2ce49f26d081)

![Yarn Contact us validation](https://github.com/user-attachments/assets/78fd306f-985d-46ac-9d21-56e2b4b98431)

yatulearn commented:
is it working ? I mean if someone enters the details then the details are going to store at backend or it is a simple page

yatulearn commented:
add the social media links of YATU Learn present in footer and shift the contact us section in about us or make a separate page for it

AbhijitMotekar99 commented:
For now it's just a contact us page with validation. For backend I will create new issue

AbhijitMotekar99 commented:
> add the social media links of YATU Learn present in footer and shift the contact us section in about us or make a separate page for it

Ok I will do it.

AbhijitMotekar99 commented:
Hey @yatulearn 
I have shifted the "Contact Us" section into the "About Us" section And also added the links all changes are Done.

AbhijitMotekar99 commented:
Hey @yatulearn

yatulearn commented:
ok

AbhijitMotekar99 commented:
Hey @yatulearn 
Please merge this PR as it's blocking me from working on other issues.

AbhijitMotekar99 commented:
Hey @yatulearn 
I resolved all conflicts now you can merge.

AbhijitMotekar99 commented:
> your pr has conflicts kindly resolve them

Hey @yatulearn 
Done

AbhijitMotekar99 commented:
Hey @yatulearn

AbhijitMotekar99 commented:
> Hey @yatulearn 
> Please merge this PR as it's blocking me from working on other issues.

Please merge this pr

AbhijitMotekar99 commented:
> Hey @yatulearn 
> Please merge this PR as it's blocking me from working on other issues.


Hey @yatulearn 
Please merge this pr

AbhijitMotekar99 commented:
Hey @yatulearn 
Please merge it

yatulearn commented:
your pr has conflicts kindly resolve them

AbhijitMotekar99 commented:
Hey @yatulearn 
I resolved all conflict please Merge this PR.

AbhijitMotekar99 commented:
Hey @yatulearn 
Please try to merge it as soon as possible because I have to work on another issues.
Thank you for understanding!

yatulearn commented:
remove the separate contact us page from header and not to make any changes in the about page except to add the contact us section just above the faqs rest leave the about page as the original you can check it on the website

AbhijitMotekar99 commented:
Hey @yatulearn 
Ok, With new changes I will raise new PR.

AbhijitMotekar99 commented:
Hey @yatulearn 
Now I done All changes which you required I removed contact us from header and added it above FAQ's.
Please merge this PR.



https://github.com/user-attachments/assets/9d8fb26e-786b-4dd5-b027-b32275e5045e

AbhijitMotekar99 commented:
Hey @yatulearn 
Please merge this PR

AbhijitMotekar99 commented:
Hey @yatulearn

AbhijitMotekar99 commented:
Hey @yatulearn 
Please merge this PR
